howdoi/howdoi.py

```diff
@@ -341,9 +346,8 @@ def _get_instructions(args):
 
     answers = []
     initial_position = args['pos']
-    spliter_length = 80
-    answer_spliter = '\n' + '=' * spliter_length + '\n\n'
-
+    answer_spliter = build_splitter('=',80)
```

Comment:
Please run `flake8` on this file as there are a couple errors. Needs a space after the comma, and also two lines before each function.